pkgs/development/python-modules/aiobtclientapi/default.nix

```diff
@@ -0,0 +1,57 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitea,
+  aiobtclientrpc,
+  async-timeout,
+  httpx,
+  torf,
+  pytest-asyncio,
+  pytest-mock,
+  pytestCheckHook,
+  setuptools,
+}:
+
+buildPythonPackage rec {
+  pname = "aiobtclientapi";
+  version = "2.0.0-unstable-2025-03-08";
```

Comment:
Wrong commit title.

Also, I contacted the author who advised staying away from the 2.0.0 version for now, it is not thoroughly tested.